sample_time: 0.03
rho_gains: [0.9, 0, 0]
alpha_gains: [1.5, 0, 0]
beta_gains: [0.1, 0, 0]

v_max: 1
v_min: -1
w_max: 3.14
w_min: -3.14


lowPassCoeff: 0.5
state_topic: "/roomba20/odometry/filtered"
cmd_topic: "/roomba20/cmd_vel"
goal_topic: "move_base_simple/goal"